test/integration/test_config_defaults.py

```diff
@@ -178,6 +178,7 @@ def expected_default_config_dir(value):
     'id_secret',  # broken: default overridden
     'job_config',  # no obvious testable defaults
     'job_config_file',  # broken: remove 'config/' prefix from schema
+    'job_metrics_config_file',
```

Comment:
Need a comment for why this doesn't work yet?

Reply:
I was considering that. That whole file deserves a TODO comment - I'm working on it this week. Let's leave as is - I'll have an update for that file within a few days.